src/qml/qgismobileapp.qml

```diff
@@ -1903,7 +1965,7 @@ ApplicationWindow {
           radius: width / 2
 
           border.width: 1.5
-          border.color: 'white'
+          border.color: "white"
```

Comment:
Just to confirm this was not overlooked, but it should remain harcoded color?

Reply:
@suricactus , correct, I just wanted to get rid of single quotes there.